CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ option(LLAMA_BUILD_SERVER   "llama: build server example" ${LLAMA_STANDALONE})
 
 # 3rd party libs
 option(LLAMA_CURL "llama: use libcurl to download model from an URL" OFF)
+option(LLAMA_LLGUIDANCE "llama: build LLGuidance library for structured output" OFF)
```

Comment:
This should somehow be indicated that it is used by the `common` library. I am not sure what would be the best way. Maybe rename it to `LLAMA_COMMON_LLGUIDANCE`? But even if we leave it like this, it's ok - just making a note.

Reply:
I re-worded the help string, maybe that is enough for now?